frontend/src/js/store/organizationSlice/index.ts

```diff
@@ -87,6 +87,9 @@ export const organizationSlice = createSlice({
     setOrganization: (state, action) => {
       state.organization = { ...state.organization, ...action.payload };
     },
+    setBillingProfile: (state, action) => {
+      state.organization = { ...state.organization, billing_profile: action.payload };
```

Comment:
```suggestion
      state.organization.billing_profile = action.payload;
```
should be possible thanks to immer